app/scripts/models/reliers/oauth.js

```diff
@@ -15,9 +15,10 @@ define([
   'lib/promise',
   'lib/oauth-errors',
   'lib/relier-keys',
-  'lib/url'
-], function (_, Relier, ResumeToken, p, OAuthErrors, RelierKeys, Url) {
-  var RELIER_FIELDS_IN_RESUME_TOKEN = ['state'];
+  'lib/url',
+  'lib/constants'
+], function (_, Relier, ResumeToken, p, OAuthErrors, RelierKeys, Url, Constants) {
+  var RELIER_FIELDS_IN_RESUME_TOKEN = ['state', 'verificationRedirect'];
```

Comment:
For a separate PR: I don't think there is any reason for `state` to exist in the resume token anymore. We realized that if reliers are using the state token for CSRF protection, a user who returns to the relier with an oauth code in a second browser should fail CSRF checks.